webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
+const HTmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = (env) => {
+  return {
+    mode: env.WEBPACK_SERVE ? "development" : "production",
```

Comment:
Seems like you are not providing `WEBPACK_SERVE` anywhere. Also, `WEBPACK_SERVE` is probably a poor naming for something like target environment, or build mode. Please, actually use an env variable.

Reply:
here env variable has WEBPACK_SERVE property which has true value in development mode, so using this env.WEBPACK_SERVE ? "development" : "production", we can dedice development or production

Reply:
Can you share the source of this information? Like maybe a link to the docs where you read it? From the first glance however, this looks like it is related to webpack-dev-server, and is set when webpac-dev-server is run. If that's the case, I don't think using this env variable to decide webpack's build mode is a good idea. Probably the most important reason why, is that running a webpack-dev-server is not the only time when we would like a development build. Like, we potentially could even use webpack's watch mode for development, and this config would not work as expected for that.

Reply:
Fixed